src/station_cmd.cpp

```diff
@@ -3222,7 +3222,7 @@ static void GetTileDesc_Station(TileIndex tile, TileDesc *td)
 		case STATION_OILRIG: {
 			const Industry *i = Station::GetByTile(tile)->industry;
 			const IndustrySpec *is = GetIndustrySpec(i->type);
-			td->owner[0] = i->owner;
+			td->owner[0] = OWNER_NONE;
```

Comment:
> I also decided to remove the unused owner field in the industry structure as it's somewhat related.

Do I understand you correctly that it has no effect on the PR if you would not do this? As when reading the patch I was confused by this change, and I do not really see why/how this belongs here?
If I understand you correct, and you are just cleaning up unused stuff, I would prefer this being in its own commit at least, but honestly .. if that is the case, I would not do it at all in this PR. A new PR to clean up these things would be better, as that means this change doesn't distract from the feature itself :)